Event/RouterGenerateEvent.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Routing\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+class RouterGenerateEvent extends Event
```

Comment:
can you please update the phpdoc for this class? add a class description that says that name, parameters and absolute are as with route generation method and add a line `@see GeneratorInterface::generate()`.

can you also add short descriptions to the properties, getters and setters? (the getters and setters mainly for IDEs showing the phpdoc to the use)